test/functional/Java8andUp/src/org/openj9/test/java/lang/Test_Thread.java

```diff
@@ -163,6 +163,7 @@ public void test_Constructor2() {
 			ct = new Thread(new SimpleThread(10));
 			ct.start();
 		} catch (Exception e) {
+			e.printStackTrace();
 			AssertJUnit.assertTrue("Failed to create subthread", false);
```

Comment:
This sequence might better be handled by:
```
    Assert.fail("Failed to create subthread", e);
```
See [org.testng.Assert#fail](https://www.javadoc.io/static/org.testng/testng/6.8.1/org/testng/Assert.html#fail(java.lang.String,%20java.lang.Throwable))

Reply:
Updated